### PR TITLE
Lab 02

### PR DESCRIPTION
**Pull request by colinandrus**
Using 3 not linearly separable parametric curves, we worked through an example of a classification supervised learning problem using a neural network.  Since solely linear functions cannot warp data in high dimensional space, non-linear activation functions are necessary to separate data in our example.  We then discussed input data structure including the one-hot encoding of the class vector.  Lastly, with using a  3 layer network with 2d input, 100d hidden size and 3d output we worked through a forward pass calculating loss and backpropagation to find gradient for gradient descent.

Lab 02 2/5/19
Colin Andrus , Aakriti Gupta, Daniel Rivera

### CODE REVIEW

**Comment by Atcold** [comment]
Okay, editing this myself.

**Comment by Atcold** [comment]
Why are there compilation errors?
screenshot 2019-02-14 14 11 08

**Comment by Atcold** [comment]
Fixed...

**Comment by colinandrus** [comment]
Looked back on overleaf and removing the empty line 141 fixed the error, sorry about that!
